js/pages/characterizations/components/characterizations/characterization-view-edit.js

```diff
@@ -84,6 +84,7 @@ define([
                 designDirtyFlag: this.designDirtyFlag,
                 areStratasNamesEmpty: this.areStratasNamesEmpty,
                 duplicatedStrataNames: this.duplicatedStrataNames,
+                isEditPermitted: this.isEditPermitted,
```

Comment:
this.isEditPermitted is boolean, so you got error when trying to get columns (context.isEditPermitted is not a function)